chain/actors/manifest.go

```diff
@@ -0,0 +1,72 @@
+package actors
+
+import (
+	"context"
+
+	"golang.org/x/xerrors"
+
+	cid "github.com/ipfs/go-cid"
+	cbor "github.com/ipfs/go-ipld-cbor"
+
+	"github.com/filecoin-project/lotus/chain/actors/adt"
+	"github.com/filecoin-project/specs-actors/v8/actors/builtin/manifest"
+)
+
+var ManifestCids map[Version]cid.Cid = map[Version]cid.Cid{
+	// TODO fill in manifest CIDs for v8 and upwards once these are fixed
+}
+
+var manifests map[Version]*manifest.Manifest
+var actorMeta map[cid.Cid]actorEntry
+
+type actorEntry struct {
+	name    string
+	version Version
+}
+
+func LoadManifests(ctx context.Context, store cbor.IpldStore) error {
```

Comment:
I don't love globals, but I don't see any good way to get rid of this.

Note that LoadManifests can be called from multiple node constructors in parallel in integration tests, so prooobably want to wrap this in global `sync.Once`..

Reply:
ok, will wrap it in a Once.

Reply:
done.